aspnetcore/fundamentals/configuration/index.md

```diff
@@ -26,37 +26,83 @@ Configuration in ASP.NET Core is performed using one or more [configuration prov
+<a name="hi2low"></a>
+
+[WebApplication.CreateBuilder](xref:Microsoft.AspNetCore.Builder.WebApplication.CreateBuilder%2A) initializes a new instance of the <xref:Microsoft.AspNetCore.Builder.WebApplicationBuilder> class with preconfigured defaults. The initialized `WebApplicationBuilder` (`builder`) provides default configuration for the app in the following order, from highest to lowest priority:
 
-1. [ChainedConfigurationProvider](xref:Microsoft.Extensions.Configuration.ChainedConfigurationSource) :  Adds an existing `IConfiguration` as a source. In the default configuration case, adds the [host](#hvac) configuration and setting it as the first source for the *app* configuration.
-1. [appsettings.json](#appsettingsjson) using the [JSON configuration provider](#file-configuration-provider).
-1. `appsettings.{Environment}.json` using the [JSON configuration provider](#file-configuration-provider). For example, `appsettings.Production.json` and `appsettings.Development.json`.
-1. [App secrets](xref:security/app-secrets) when the app runs in the `Development` environment.
-1. Environment variables using the [Environment Variables configuration provider](#evcp).
 1. Command-line arguments using the [Command-line configuration provider](#command-line).
+1. Non-prefixed environment variables using the [Non-prefixed environment variables configuration provider](#evcp).
+1. [User secrets](xref:security/app-secrets) when the app runs in the `Development` environment.
+1. `appsettings.{Environment}.json` using the [JSON configuration provider](#file-configuration-provider). For example, `appsettings.Production.json` and `appsettings.Development.json`.
+1. [appsettings.json](#appsettingsjson) using the [JSON configuration provider](#file-configuration-provider).
+1. A fallback to the host configuration described in the [next section](#host).
 
-Configuration providers that are added later override previous key settings. For example, if `MyKey` is set in both `appsettings.json` and the environment, the environment value is used. Using the default configuration providers, the  [Command-line configuration provider](#clcp) overrides all other providers.
+<a name="host"></a>
 
-For more information on `CreateBuilder`, see [Default builder settings](xref:fundamentals/host/generic-host#default-builder-settings).
+### Default host configuration sources
+
+The following list contains the default host configuration sources from highest to lowest priority:
+
+1. [`ASPNETCORE_` prefixed environment variable](xref:fundamentals/environments)
+1.  Command-line arguments using the [Command-line configuration provider](#command-line)
+1. [`DOTNET_` prefixed environment variables](/dotnet/core/tools/dotnet-environment-variables)
```

Comment:
It looks like my last comment here is outdated. I'll redo my suggestion for reasons explained in https://github.com/dotnet/AspNetCore.Docs/pull/25686/files#r868567158. Basically, these link to 

```suggestion
1. `ASPNETCORE_`-prefixed environment variables using the [Environment variables configuration provider](xref:Microsoft.Extensions.Configuration.EnvironmentVariables.EnvironmentVariablesConfigurationProvider).
1.  Command-line arguments using the [Command-line configuration provider](#command-line)
1. `DOTNET_`-prefixed environment variables using the [Environment variables configuration provider](xref:Microsoft.Extensions.Configuration.EnvironmentVariables.EnvironmentVariablesConfigurationProvider).
```

If we don't like linking directly to `EnvironmentVariablesConfigurationProvider` here, we could have `#evcp` discuss all environment variables, not just non-prefixed environment variables. It's already been updated to at least mention the prefixed environment variables.